docs_src/design/adr/013-Device-Service-Events-Message-Bus.md

```diff
@@ -0,0 +1,215 @@
+# Device Services Send Events via Message Bus
+
+  * [Status](#status)
+  * [Context](#context)
+  * [Decision](#decision)
+    + [Which Message Bus implementations?](#which-message-bus-implementations-)
+    + [Go Device SDK](#go-device-sdk)
+    + [C Device SDK](#c-device-sdk)
+    + [Core Data and Persistence](#core-data-and-persistence)
+    + [V2 Event DTO](#v2-event-dto)
+    + [Message Envelope](#message-envelope)
+    + [Redis Streams](#redis-streams)
+    + [Application Services](#application-services)
+    + [Message Bus Topics](#message-bus-topics)
+    + [Configuration](#configuration)
+      - [Device Services](#device-services)
+        * [[MessageQueue]](#-messagequeue-)
+      - [Core Data](#core-data)
+        * [[MessageQueue]](#-messagequeue--1)
+      - [Application Services](#application-services-1)
+        * [[MessageBus]](#-messagebus-)
+        * [[Binding]](#-binding-)
+    + [Secure Connections](#secure-connections)
+  * [Consequences](#consequences)
+
+## Status
+
+**proposed**
+
+## Context
+
+This ADR details how device services will send EdgeX Events to other services via the EdgeX Message Bus. This is instead of how they currently send the Events via HTTP to Core Data, which then puts the Events on the Message Bus. 
```

Comment:
How about reverse this so that it reads better:

EdgeX currently ....
This ADR proposed to change this to deliver events to the message bus directly.

Reply:
Done